crates/utils/src/utils/validation.rs

```diff
@@ -82,14 +86,83 @@ pub fn is_valid_body_field(body: &Option<String>) -> LemmyResult<()> {
 }
 
 pub fn is_valid_bio_field(bio: &str) -> LemmyResult<()> {
-  let check = bio.chars().count() <= BIO_MAX_LENGTH;
-  if !check {
-    Err(LemmyError::from_message("bio_length_overflow"))
+  max_length_check(bio, BIO_MAX_LENGTH, String::from("bio_length_overflow"))
+}
+
+/// Checks the site name length, the limit as defined in the DB.
+pub fn site_name_length_check(name: &str) -> LemmyResult<()> {
+  min_max_length_check(
```

Comment:
I noticed that the site name is required in the database, so I added a check to also validate a minimum site name length. Otherwise the changes are just moved from when it lived in `site_utils.rs`.

Reply:
Good catch, thanks.